pymetis/__init__.py

```diff
@@ -199,3 +199,43 @@ def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
                       eweights, options, recursive)
 
 # vim: foldmethod=marker
+
+
+def part_mesh(nParts, connectivity, options=None):
+    """This function is used to partition a mesh into `nParts` parts based on a
+    partitioning of the mesh's nodal graph.
```

Comment:
Describe what a mesh is, and be more precise when describing the data structure.